cmd/exporter/main.go

```diff
@@ -63,6 +65,7 @@ func init() {
 	cmd.FlagLogLevel(&logLevel)
 	cmd.FlagCacheImage(&cacheImageTag)
 	cmd.FlagCacheDir(&cacheDir)
+	cmd.FlagPlatformDir(&platformDir)
```

Comment:
Apologies, I know these changes are necessary b/c I originally pushed you in the wrong direction, but if we remove this unnescessary `-platform` flag on exporter and put the `-project-metadata` flag back we will be good to merge

Reply:
Good spot; this should now be resolved.  Thanks @ekcasey